codingtovictory/day6/takeHomeChallenge/index.html

```diff
@@ -0,0 +1,25 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="FirstA">
+    <a href=https://www.youtube.com/watch?v=--khbXchTeE">Checkout this GPT-4 video.</a>
+    </div>
+
+    <div class="SecondA" >
+    <a href="Redirect.html">Click here to be taken to our other super secret page.</a>
```

Comment:
Hey, you did great with the internal link and linking another html file to your home page